### PR TITLE
feat: #424 Add progress bar to storybook

### PR DESCRIPTION
**Pull request by Alex89198900**
1. Create story
2. center text in the progress bar

### CODE REVIEW

**Review by tritonJS826 — APPROVED** [review]

File: mw-webapp/src/component/progressBar/ProgressBar.module.scss [hunk 1]
```diff
@@ -19,7 +20,11 @@ $progressBarBorderRadius: 20px;
 }
 
 .progressIndicatorPercent {
+  position: absolute;
+  right: 0;
+  left: 0;
```
Comment (tritonJS826) [inline mw-webapp/src/component/progressBar/ProgressBar.module.scss:25]: how does it works at the same time?)

**Review by tritonJS826 — CHANGES_REQUESTED** [review]

File: mw-webapp/src/component/progressBar/ProgressBar.module.scss (hunk 1 above)
Comment (tritonJS826) [inline mw-webapp/src/component/progressBar/ProgressBar.module.scss:25]: why not to handle positioning with flexboxes? Flexbox css has pretty good interface

File: mw-webapp/src/component/progressBar/ProgressBar.module.scss [hunk 2]
```diff
@@ -19,7 +20,11 @@ $progressBarBorderRadius: 20px;
...
   margin-right: $marginSmall;
   margin-left: $marginSmall;
   color: var(--primaryTextColor);
+  text-align: center;
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/progressBar/ProgressBar.module.scss:29]: Please check do we need this property)

Comment (tritonJS826) [inline mw-webapp/src/component/progressBar/ProgressBar.module.scss:29]: text should be placed in the center of the progress bar. Does it works now this way?

File: mw-webapp/src/component/progressBar/ProgressBar.module.scss (hunk 1 above)
Comment (Alex89198900) [reply mw-webapp/src/component/progressBar/ProgressBar.module.scss:25]: I can't explain exactly how it works). But this bundle makes it possible to set text-align: center.

Comment (tritonJS826) [reply mw-webapp/src/component/progressBar/ProgressBar.module.scss:25]: Hahah) ok I see

**Review by Ekaterina1994 — COMMENTED** [review]
PLease rename PR accorfint to our docs

File: mw-webapp/src/component/progressBar/ProgressBar.module.scss (hunk 1 above)
Comment (tritonJS826) [reply mw-webapp/src/component/progressBar/ProgressBar.module.scss:25]: so why not to use flexboxes?

**Review by tritonJS826 — APPROVED** [review]

**Review by Ekaterina1994 — APPROVED** [review]

**Pull request merged**